examples/foundational/19-openai-realtime.py

```diff
@@ -198,6 +216,16 @@ async def on_client_connected(transport, client):
         # Kick off the conversation.
         await task.queue_frames([LLMRunFrame()])
 
+        async def set_tools_after_delay():
+            await asyncio.sleep(15)
+            new_tools = ToolsSchema(
+                standard_tools=[weather_function, restaurant_function, get_news_function]
+            )
+            logger.info("Registering new tool with LLMSetToolsFrame")
+            await task.queue_frames([LLMSetToolsFrame(tools=new_tools)])
+
+        asyncio.create_task(set_tools_after_delay())
```

Comment:
We don't really need a task, because `on_client_connected` is already a task. So we cna jsut sleep and queue the frames. If we want a new task we should `await` it.